atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -150,6 +150,9 @@ def run_in_orchestrator(self):
               replaced: <bool>  # plugin modified pullspec (repo/registry/tag changed)
             - original: ........
           created_by_osbs: <bool>
+        SIDE-EFFECT:
```

Comment:
Orchestrator cannot replace data in CSV, that's happening on worker

Reply:
@MartinBasti; So I should be calling `_replace_other_metadata_from_csv_modifications()` from `run_in_worker()` instead?

Reply:
run_in_worker() inside 'if not operator_csv.has_related_images()'

when has_releated_images is there freshmaker shouldn't be rebuilding images

Reply:
@rcerven; ACK exactly what I needed.